models/classes/session/source/PortalSessionSourceMatcher.php

```
@@ -0,0 +1,37 @@
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoLti\models\classes\session\source;
+
+use common_session_Session;
+use oat\tao\model\session\Context\TenantDataSessionContext;
+use oat\tao\model\session\source\SessionSourceMatcherInterface;
+use oat\taoLti\models\classes\TaoLtiSession;
+
+class PortalSessionSourceMatcher implements SessionSourceMatcherInterface
```

PHPStan (2.2.7)

[error] Class oat\taoLti\models\classes\session\source\PortalSessionSourceMatcher implements unknown interface oat\tao\model\session\source\SessionSourceMatcherInterface.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(interface.notFound)